scripts/start-relayer.sh

```diff
@@ -1,33 +1,48 @@
 #!/bin/bash
 
-RELAY_PATH="${RELAY_PATH:-local_testnet}"
+RELAY_PATH="${RELAY_PATH:=local_local2}"
+RELAY_CMD="${RELAY_CMD:-rly}"
+RELAYER_HOME="${RELAYER_HOME:-$HOME/.relayer}"
 
 # We have to get CHAIN_1 and CHAIN_2 by splitting the path
-CHAIN_1=$(echo "${RELAY_PATH}" | cut -d "_" -f 1)
-CHAIN_2=$(echo "${RELAY_PATH}" | cut -d "_" -f 2)
+CHAIN_1=$(echo "$RELAY_PATH" | cut -d "_" -f 1)
+CHAIN_2=$(echo "$RELAY_PATH" | cut -d "_" -f 2)
 
 check_keys() {
     CHAIN=$1
-    rly keys show $CHAIN &> /dev/null
+    "$RELAY_CMD" keys show "$CHAIN" --home "$RELAYER_HOME" &> /dev/null
     status=$?
-    if [ $status != 0 ]; then
+    if [ "$status" != 0 ]; then
         echo "No keys exist for $CHAIN" >&2
-        echo "Consider using an existing key with 'rly keys restore $CHAIN default \"mneomnic\"'" >&2
-        echo "Alternatively, if an account does not exist you can create one with 'rly keys add $CHAIN default' and then funding it" >&2
+        echo "Consider using an existing key with '$RELAY_CMD keys restore $CHAIN default \"mneomnic\" --home $RELAYER_HOME'" >&2
+        echo "Alternatively, if an account does not exist you can create one with '$RELAY_CMD keys add $CHAIN default --home $RELAYER_HOME' and then funding it" >&2
     fi
-    return $status
+    return "$status"
 }
 
-check_keys ${CHAIN_1}
-if [ $? != 0 ]; then
-    echo "Cannot start relayer until ${CHAIN_1} has a key"
+check_links() {
+    PATH_STATUS=$("$RELAY_CMD" paths list --home "$RELAYER_HOME" | grep "$RELAY_PATH")
+    PASSING=$(grep -o "✔" <<< "$PATH_STATUS" | wc -l | tr -d ' ')
```

Comment:
It's best practice to quote command output when you're saving it to a variable. It can save your bacon in some weird situations and also helps prevent injection attacks.

```suggestion
    PATH_STATUS="$( "$RELAY_CMD" paths list --home "$RELAYER_HOME" | grep "$RELAY_PATH" )"
    PASSING="$( grep -o "✔" <<< "$PATH_STATUS" | wc -l | tr -d ' '  )"
```

Reply:
Makes sense to me.